proto/proto/wallet.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+package penumbra.wallet;
+
+import "transaction.proto";
+import "crypto.proto";
+
+service WalletProtocol {
+  // TODO: do we want to include auth/signatures in all requests here?
+  /// Get current status of chain sync
+  rpc Status() returns (StatusResult);
+  /// Create a new transaction
+  rpc CreateTransaction(CreateTransactionRequest) returns (CreateTransactionResult);
+  // TODO: not sure what listing transactions would actually look like
+  /// List pending transactions
+  rpc Transactions(TransactionsRequest) returns (stream transaction.Transaction);
+  // TODO: is this UtxoResult analogous to transaction.Output? or UnspentNote? should we use that here?
+  /// List unspent transaction outputs
+  rpc Utxo(UtxoRequest) returns (stream UtxoResult);
+  /// Display current wallet balance
+  rpc Balance(BalanceRequest) returns (stream BalanceResult);
+}
+
+// Requests the wallet server to sign and submit a transaction.
+message CreateTransactionRequest {
+    transaction.TransactionBody body = 1;
+}
+
+// Returns whether the creation of the transaction was successful or not.
+message CreateTransactionResult {
+    bool success = 1;
+}
+
+// Returns the status of the wallet server and whether it is synchronized with the chain state.
+message StatusResult {
+  /// Whether the wallet service is synchronized with the chain state.
+  bool synchronized = 1;
+}
```

Comment:
It might be handy to return heights, like
```
// The height reported by the full node
uint64 chain_height = 2;
// The height the wallet has synchronized to so far
uint64 sync_height = 3;
```
so that the client would be able to get progress info.